packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -294,4 +331,62 @@ contract SystemConfig is OwnableUpgradeable, Semver {
 
         _resourceConfig = _config;
     }
+
+    /**
+     * @notice An external setter for the `attestorSet` mapping. This method is used to
+     *         authenticate or deauthenticate an attestor in the `AttestationDisputeGame`.
+     *
+     * @param _attestor Address of the attestor to authenticate or deauthenticate.
+     * @param _authenticated True if the attestor should be authenticated, false if the
+     *        attestor should be removed.
+     */
+    function setAttestor(address _attestor, bool _authenticated) external onlyOwner {
+        _setAttestor(_attestor, _authenticated);
+        emit AttestorSetUpdate(_attestor, _authenticated);
+    }
+
+    /**
+     * @notice An internal setter for the attestor set.
+     *
+     * @param _attestor Address of the attestor to authenticate or deauthenticate.
+     * @param _authenticated True if the attestor should be authenticated, false if the
+     *        attestor should be removed.
+
+     */
+    function _setAttestor(address _attestor, bool _authenticated) internal {
+        uint256 len = _attestorSet.length;
+        for (uint256 i = 0; i < len; i++) {
+            if (_attestorSet[i] == _attestor) {
+                if (_authenticated) {
+                    revert("SystemConfig: attestor already authenticated");
+                } else {
+                    // Remove the attestor from the array by swapping it with
+                    // the last attestor and then popping the last element.
+                    _attestorSet[i] = _attestorSet[len - 1];
+                    _attestorSet.pop();
+                    return;
+                }
+            }
+        }
+        if (_authenticated) {
+            _attestorSet.push(_attestor);
+        }
+    }
+
+    /**
+     * @notice An external setter for the `attestationThreshold` variable.
+     *         This method is used to set the number of signatures required
+     *         to invalidate an output proposal in the `AttestationDisputeGame`.
+     *
+     * @param _attestationThreshold The new attestation threshold.
+     */
+    function setAttestationThreshold(uint256 _attestationThreshold) external onlyOwner {
```

Comment:
I think it may be easier if the threshold was a % and not an absolute number. If its an absolute number then we need to manage updating it as new attestors are added

Reply:
Makes sense to me!